libs/langchain/langchain/utilities/tensorflow_datasets.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Callable, Dict, Iterator, List, Optional
+
+from pydantic import BaseModel, root_validator
+
+from langchain.schema import Document
+
+logger = logging.getLogger(__name__)
+
+
+class TensorflowDatasets(BaseModel):
```

Comment:
i think we should just put all this log on the document loader, not being used anywhere else at the moment

Reply:
I thought about it. 
It is probably too stretching, the TF Datasets has many datasets that can be used as tools, especially for the model experimentations. That was a reason to generalize it into a utility.